tests/index.html

```diff
@@ -9,26 +9,35 @@
 <div id="mocha"></div>
 <script src="../bower_components/chai/chai.js"></script>
```

Comment:
Instead of including these files directly and declaring globals, you should allow requirejs to include them. Whenever they are needed by a test module, include the dependency in the list of dependencies.

You'll probably have to declare the globals files like `chai` and `mocha` create. For an example, see [how the content server does it](https://github.com/mozilla/fxa-content-server/blob/c39d365ef4eb734777d947d6bf4e20b578a096cb/app/scripts/require_config.js#L45).